desktop/js/jMQTT.js

```diff
@@ -809,6 +809,7 @@ function addCmdToTable(_cmd) {
 		tr += '</td><td>';
         tr += '<span><label class="checkbox-inline"><input type="checkbox" class="cmdAttr checkbox-inline" data-l1key="isVisible" checked/>{{Afficher}}</label></span><br> ';
         tr += '<span><label class="checkbox-inline"><input type="checkbox" class="cmdAttr checkbox-inline" data-l1key="configuration" data-l2key="retain"/>{{Retain}}</label></span><br> ';
+        tr += '<span><label class="checkbox-inline"><input type="checkbox" class="cmdAttr checkbox-inline" data-l1key="configuration" data-l2key="auto_publish"/>{{Pub. auto}} <sup><i class="fa fa-question-circle tooltips" title="Publication automatique en MQTT lors d\'un changement <br>(Utiliser avec au moins une commande info dans Valeur)."></i></sup></label></span><br> ';
```

Comment:
why "auto_publish"? most settings name are in camelCase (I know it's a mess even in jeedom Core names...)

Reply:
ok auto_publish -> autoPub